packages/wonder-blocks-dropdown/src/components/dropdown-core.js

```diff
@@ -301,16 +291,7 @@ class DropdownCore extends React.Component<Props, State> {
                     this.focusedIndex = 0;
                     // Reset the knowlege that things had been clicked
                     this.itemsClicked = false;
-                    if (this.keyboardNavOn) {
-                        // If keyboard navigation was already on, use that
-                        this.scheduleToFocusCurrentItem();
-                    } else {
-                        // Otherwise shift focus to the original focus item
-                        // (the opener) to listen for further keyboard events
-                        if (this.props.openerElement) {
-                            this.props.openerElement.focus();
-                        }
-                    }
+                    this.scheduleToFocusCurrentItem();
```

Comment:
I thought that using the scroll wheel would cause a new item to be focus, but scrolling causes the focused item to become unfocused.  This is the same behaviour as before this PR so not an issue for this review.  I'm not sure if we even care about the interaction between using the keyboard and the mouse at the same time.  It seems likely that most people only use one at a time.